lib/hexo/post.ts

```diff
@@ -536,9 +538,9 @@ class Post {
           data.content = cacheObj.restoreAllSwigTags(content);
 
           // Return content after replace the placeholders
-          if (disableNunjucks) return data.content;
+          if (disableNunjucks || !hasTag) return data.content;
```

Comment:
The conditional operator here is a bit hard to follow, but since I can't think of a better solution, I think this is fine as it is.